### PR TITLE
added permission denied info

### PR DESCRIPTION
**Pull request by tiruworktito**
**When you open your pull requests, remove the lines that start and end with underscores (\_)**

## Purpose
look at my standard practice

## Approach
added the link to the readme.md

## Pre-Testing TODOs
make sure to test the correct link 

## Learning
_Describe the research stage_

_Links to blog posts, patterns, libraries or addons used to solve this problem_

_If this closes an issue, name it here. If it doesn't, remove this line_
Closes #6

### CODE REVIEW

**Review by fedluk70 — COMMENTED** [review]
nice job

File: README.md [hunk 1]
```diff
@@ -3,6 +3,12 @@
 
 ## Contributing to this Repository
 
+* Error: Permission denied (publickey info
+https://docs.github.com/en/authentication/troubleshooting-ssh/error-permission-denied-publickey
+
 #### You are expected to contribute _something_ to our processes. You can do this in many different ways, such as addng documentation that helped you out, writing up a markdown sheet for this repository on a topic you are passionate about, leading a workshop, or posting discussion topics in Bitwise's #devops-apprenticeship-attachment channel.
 
+* this is my six pr
```
Comment (michaelachrisco) [inline README.md]: Lets remove any test text within the PR.

File: README.md [hunk 2]
```diff
@@ -3,6 +3,12 @@
 
 ## Contributing to this Repository
 
+* Error: Permission denied (publickey info
```
Comment (michaelachrisco) [inline README.md:8]: Nice, great link. I remember we had this issue before. What do you think of adding back the parenthesis here to make the verbiage consistent: 
```suggestion
* Error: Permission denied (publickey info)
```

**Review by michaelachrisco — APPROVED** [review]
Awesome job @tiruworktito! looks good, any links that help out with debugging are a great resource. One non-blocking suggestion for the Pull Request. 

Thanks for the contribution!

**Comment by tiruworktito** [comment]
conflict resolve

**Pull request merged**